build.gradle

```
@@ -262,25 +262,25 @@ allprojects {
                             substitute module('org.labkey:labkey-client-api') with project(BuildUtils.getRemoteApiProjectPath(gradle))
```

Does the `labkey-client-api` substitution need to be updated?
```suggestion
                            substitute module('org.labkey:labkey-client-api') using project(BuildUtils.getRemoteApiProjectPath(gradle))
```

---

Yes, thanks. Wasn't building that locally.